konfera/templates/konfera/meetups.html

```diff
@@ -3,40 +3,42 @@
 
 
 {% block title %}
-    {% trans "Meetups List" %}
+{% trans "Meetups List" %}
 {% endblock %}
 
 
 {% block content %}
-    <div class="list-group">
-        {% for meetup in meetups %}
-            <a href="{% url 'event_details' meetup.slug %}" class="list-group-item">
-                <h4 class="list-group-item-heading"> {{ meetup.title }} </h4>
-                <p class="list-group-item-text">{{ meetup.date_from|date:"d. F Y – H:m"}} - {{ meetup.location}}</p>
-            </a>
-        {%  endfor %}
-    </div>
-
-    <nav aria-label="Page navigation">
-        <ul class="pagination">
-            {% if meetups.has_previous %}
-                <li>
-                    <a href="?page={{ meetups.previous_page_number }}" aria-label="Previous" >
-                    <span aria-hidden="true">&laquo;</span></a>
-                </li>
-            {% endif %}
-
-            {% for page in meetups.paginator.page_range %}
-                <li><a href="?page={{ page }}"> {{ page }} </a></li>
-            {% endfor %}
-
-            {% if meetups.has_next %}
-                <li>
-                    <a href="?page={{ meetups.next_page_number }}" aria-label="Next" >
-                    <span aria-hidden="true">&raquo;</span></a>
-                </li>
-            {% endif %}
-        </ul>
-    </nav>
+<div class="list-group">
+    {% for meetup in object_list %}
+    <a href="{% url 'event_details' meetup.slug %}" class="list-group-item">
+        <h4 class="list-group-item-heading"> {{ meetup.title }} </h4>
+        <p class="list-group-item-text">{{ meetup.date_from|date:"d. F Y – H:m"}} - {{ meetup.location}}</p>
```

Comment:
Display date and time from - to (in case event ends the same day), in case event end on different date just show from to dates.
please add empty space before }}